adapters/quick/src/main/scala/caliban/QuickRequestHandler.scala

```diff
@@ -25,6 +25,10 @@ final private class QuickRequestHandler[R](
   import QuickRequestHandler._
   import ValueJsoniter.stringListCodec
 
+  private val readerConfig: ReaderConfig = ReaderConfig
+    .withMaxBufSize(2147483645)
+    .withMaxCharBufSize(2147483645)
```

Comment:
Probably good to also add `.withAppendHexDumpToParseException(false)` (for consistency and UX)